entrypoint.sh

```diff
@@ -5,11 +5,13 @@ set -e
 
 APT_DEPENDENCIES=$1
 CODECOV_TOKEN=$2
-SCRIPT_BEFORE_CMAKE=$3
-CMAKE_ARGS=$4
-SCRIPT_BETWEEN_CMAKE_MAKE=$5
-SCRIPT_AFTER_MAKE=$6
-SCRIPT_AFTER_MAKE_TEST=$7
+CMAKE_ARGS=$3
+
+SOURCE_DEPENDENCIES=".github/ci-bionic/dependencies.yaml"
+SCRIPT_BEFORE_CMAKE="../.github/ci-bionic/before_cmake.sh"
```

Comment:
Minor nit: Just noticed that these aren't consistent.

Reply:
Yeah the reason is that the scripts are run from the `build` folder. At least it's all hidden inside this script now. It should be possible to make them consistent though.